charts/datastore/templates/deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
               port: 443
               {{ end }}
               {{ if (eq .Values.datastore.serverMode "AUTOTLS") }}
-              host: {{ .Values.datastore.autoTlsDomain }}
+              host: {{ required "In TLSAUTO mode, hostname must be specified in datastore.autoTlsDomain." .Values.datastore.autoTlsDomain  }}
```

Comment:
Why does it say TLSAUTO instead of AUTOTLS? o_o

Reply:
lol